### PR TITLE
Add rules description for UFW

### PR DESCRIPTION
**Pull request by DeWaRs1206**
### Description

Adding description in the rules for UFW

### Issues Resolved

None

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Provides a set of primitives for managing firewalls and associated rules.'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.7.0'
+version '2.7.1'
```
Comment (damacus) [inline metadata.rb:7]: Please revert this change, our auto release system will take care of this for us :)

**Comment by damacus** [comment]
Can you add a changelog entry please?

**Comment by gene1wood** [comment]
Now that #256 is merged feel free to close this PR

**Comment by DeWaRs1206** [comment]
Replaced by https://github.com/sous-chefs/firewall/pull/215
